src/scc.js

```diff
@@ -0,0 +1,82 @@
+import calculateScc from '@rtsao/scc';
+import { hashObject } from 'eslint-module-utils/hash';
+import resolve from 'eslint-module-utils/resolve';
+import ExportMapBuilder from './exportMap/builder';
+import childContext from './exportMap/childContext';
+
+let cache = new Map();
+
+export default class StronglyConnectedComponentsBuilder {
+  static clearCache() {
+    cache = new Map();
```

Comment:
```suggestion
    cache.clear();
```
this might be less GC intensive? i don't feel strongly tho

Reply:
Maybe. `clearCache` is currently used only by tests, so I think it doesn't matter.

Reply:
How do we move forward with this? Do we make the change or do we resolve this conversation as-is?